### PR TITLE
Release 0.73

### PR DESCRIPTION
**Pull request by doctrino**
Merge after #399 #397

### CODE REVIEW

File: cognite/neat/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.72.3"
+__version__ = "0.72.4"
```
Comment (nikokaoja) [inline cognite/neat/_version.py:1]: ok great, mind my comment in 399 ref version bump

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ Changes are grouped as follows:
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [0.72.4] - 17-04-24
+### Added
```
Comment (nikokaoja) [inline docs/CHANGELOG.md:19]:
```suggestion
### Added
- Proper parsing/serialization of `inf`
```

**Review by nikokaoja — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 .PHONY: run-explorer run-tests run-linters build-ui build-python build-docker run-docker compose-up
 
-version="0.72.3"
+version="0.72.4"
```
Comment (doctrino) [inline Makefile:3]:
```suggestion
version="0.73.0"
```

File: cognite/neat/_version.py (hunk 1 above)
Comment (doctrino) [inline cognite/neat/_version.py:1]:
```suggestion
__version__ = "0.73.0"
```

File: docs/CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +15,21 @@ Changes are grouped as follows:
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [0.72.4] - 17-04-24
```
Comment (doctrino) [inline docs/CHANGELOG.md:18]:
```suggestion
## [0.73.0] - 17-04-24
```

File: pyproject.toml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-neat"
-version = "0.72.3"
+version = "0.72.4"
```
Comment (doctrino) [inline pyproject.toml:3]:
```suggestion
version = "0.73.0"
```

**Pull request merged**